setup.py

```diff
@@ -130,6 +130,8 @@ def write_version_py(filename=None):
     data = read_module('__init__')
     __requires__ = data['__requires__']
     __extras_require__ = data['__extras_require__']
+    with open("README.rst", "r", encoding="utf-8") as readme:
+        LONG_DESCRIPTION=readme.read().split(".. end_of_long_description")[0]
```

Comment:
🤖  TraitsUI and Pyface don't have flake8 in CI... Flake8 would have complained about the missing spaces around the equal sign...  Not your fault - we should have the machine nagging us about this sort of thing.